### ISSUE TITLE
law suit against iota 

### ISSUE THREAD

**Issue by elvis1965isik** [iotaledger/wallet#607]
i lost a lot of money, because of an inferior product (iota wallet) thats why i am preparing a lawsuit against iota, if people are interested, and have lost money also. Send me a email at albert@mandrilvastgoed.nl

**Comment by cloggy45** [comment]
You use these things at your own risk, there is no guarantee.

**Comment by ThomasPepperz** [comment]
I think your time would be better spent recovering your coin. What is the
status of your addresss on the list? Email me Thomas.peppers@colorado.edu
and I will help.

**Comment by AwenCrypto** [comment]
I agree that this does not bode well for the company however, **growth is painful**.  This is growth.

**You are part of that growth** - WE choose to be part of that growth and so during this time, there will be pain.. but I would not worry about it.  I do not wish to be insensitive, I lost some coins myself.  Focus on clear thinking and finding a solution.

Lest we forget that the crypto environment is full of risk, including the appearance to you of the situation
with your coins.  This too shall pass.

**Comment by elvis1965isik** [comment]
thanks thomas i send you a email tommorow.

albert smit

**Comment by elvis1965isik** [comment]
Awencrypto  i work hard for my money, and growth is painful, thats nice,, but if a company like iota makes mistakes in their software, no problem, but if you make mistakes and your customers loose money ore coins ore whatsoever, part because your product is not good, than the least they can do is help you with your problem.

But nothing of this, i tried to reach everybody i could at iota via twitter email and so on, but there is no ressponse at all.  And i think this is a bad and arrogant way of treating the first adapters of your project

**Comment by dannice** [comment]
This fixed my problem:
1. update your iota wallet to the newest release
Then follow these steps:
https://forum.helloiota.com/588/Help-My-IOTA-balance-is-zero-steps-to-help-you-find-your-balance-v254?PageIndex=1

**Comment by elvis1965isik** [comment]
i also downloaded the new wallet, and after my balance was zero, i did what i had to do with the tangle, in my old wallet i had 2398 iota, after i tangled in the new wallet (after a few times) my new wallet showed 998 iotas, so i tangled again and this time i received 5 iotas but the 998 where gone.Tangled again and now there were the 998 back again, so i thought lets first send this 998 iota to bitfinex before they are gone again.So i send those to bitfinex and went tangeling again for the rest, after a few times the rest of my iotas were on my balance, again.I was afraid i would loose them again so i sended all my iotas to bitfinex.After a few hours still nothing on bitfinex?? so i tried again to send it to bitfinex, now a month later everything is pending and some transactions are confirmed but show 0 behind the transaction. And still nothing on bitfinex. If you send iotas to bitfinex and weeks later it is still pending then you should cancel your transaction, but for as i know thats not possible. so what can you do more.

**Comment by BeBlockTech** [comment]
@dannice I've done all that... Even spoke with Eric Hope on his FB page. After he "tried" to help me, the best thing he could do was to delete my comment and block me from his page. Very, very mature from someone of his age to be blocking people from a page because he can't help. It proves how much he trust this project. I asked once on Twitter, they told me to use the Western Union and also blocked me from their page.

My coins have been missing for over 100 days now and there is not one person who's able to help, the only thing I hear is "Don't worry, coins are safe..." After more than three months I don't want the coins to be safe, I want my coins in my wallet so I can get rid of them. It's very frustrating to know that devs can take custody of your tokens whenever it pleases them.

**Comment by Nullpo** [comment]
lawsuit against iota?

The license is GPL v3. Good luck with that ;).

"THERE IS NO WARRANTY FOR THE PROGRAM, [...] PROVIDE THE PROGRAM "AS IS" WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING, BUT NOT LIMITED TO,
THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
PURPOSE.  THE ENTIRE RISK AS TO THE QUALITY AND PERFORMANCE OF THE PROGRAM
IS WITH YOU.  SHOULD THE PROGRAM PROVE DEFECTIVE, YOU ASSUME THE COST OF
ALL NECESSARY SERVICING, REPAIR OR CORRECTION."

**Comment by b-f-** [comment]
So, if one releases software and creates a hype around it, it's not one's responsibility, because GPL? The fake IOTA seed generator should have been published under GPL, then, to absolve authors of responsibility? An interesting legal proposal, indeed.

**Comment by ThomasPepperz** [comment]
Liam's  advice is on mark. I've had trouble from my wallet three or four
times now and each time it has been resolved (to my financial benefit given
the continuously increasing price). Please notify someone if you're issue
has been resolved so we can close them.

**Comment by elvis1965isik** [comment]
Nullpo  Thank you for wishing me luck, 

Youre comment, , i see it different, i have a company in Germany, and we have international law in Europe, iota is registred in Germany, so here is my vision.

I bought IOTA tokens with my own money, so at that moment i am the legally owner of these tokens.
Then i put them in an open source wallet (iota) whit my own seed for safety.

And now comes the legal issue, somebody took my iota tokens out of this wallet without my permission.

And that is called theft in Germany, if this foundation sends the same iota tokens back to the wallet, you could see it as borrowing, but i dont think a judge will look at it that way.

**Comment by elvis1965isik** [comment]
And Thomas Pepperz, thanks for the help

**Comment by DJScooby3D** [comment]
i lost 39,999,5000 iota Which is 40 MI im with you elvis1965isik someone should start a class action suit

**Comment by ThomasPepperz** [comment]
You keep going from issue page to page trolling.

Dude you will be banned from github if you do not stick to technical
issues. Go to Reddit or twitter if you’re gonna troll. That address and
claims associated with it have been discredited. I am going to submit a
pull request to close this case unless you want to stick to fixing your
problem. The wallet has not been hacked. Individuals making dumb decisions
may have made themselves vulnerable.

Now, walk us through what you’ve done? Have you begun the transition or is
your complaint that you were “hacked?”
